types/chrome/index.d.ts

```diff
@@ -7184,25 +7184,50 @@ declare namespace chrome.storage {
         getBytesInUse(callback: (bytesInUse: number) => void): void;
         /**
          * Gets the amount of space (in bytes) being used by one or more items.
-         * @param keys A single key or list of keys to get the total usage for. An empty list will return 0. Pass in null to get the total usage of all of storage.
+         * @param keys Optional. A single key or list of keys to get the total usage for. An empty list will return 0. Pass in null to get the total usage of all of storage.
+         * @return The `getBytesInUse` method provides its result via callback or returned as a `Promise` (MV3 only).
+         */
+        getBytesInUse(keys?: string | string[] | null): Promise<number>;
+        /**
+         * Gets the amount of space (in bytes) being used by one or more items.
+         * @param keys Optional. A single key or list of keys to get the total usage for. An empty list will return 0. Pass in null to get the total usage of all of storage.
          * @param callback Callback with the amount of space being used by storage, or on failure (in which case runtime.lastError will be set).
          * Parameter bytesInUse: Amount of space being used in storage, in bytes.
          */
         getBytesInUse(keys: string | string[] | null, callback: (bytesInUse: number) => void): void;
+        /**
+         * Removes all items from storage.
+         * @return The `clear` method provides its result via callback or returned as a `Promise` (MV3 only).
+         */
+        clear(): Promise<void>;
         /**
          * Removes all items from storage.
          * @param callback Optional.
          * Callback on success, or on failure (in which case runtime.lastError will be set).
          */
         clear(callback?: () => void): void;
+        /**
+         * Sets multiple items.
+         * @param items An object which gives each key/value pair to update storage with. Any other key/value pairs in storage will not be affected.
+         * Primitive values such as numbers will serialize as expected. Values with a typeof "object" and "function" will typically serialize to {}, with the exception of Array (serializes as expected), Date, and Regex (serialize using their String representation).
+         * @return The `set` method provides its result via callback or returned as a `Promise` (MV3 only).
```

Comment:
I'd not mention the callback here, since this definition doesn't have it. You could shorten this to `@return A void Promise`, but it's up to you.